setup.py

```diff
@@ -25,7 +25,7 @@ def run_tests(self):
     'python-dateutil >= 2.5, <3.0',
     'click >= 6.6, <7.0',
     'czech-holidays',
-    'blessings >= 1.6, < 2.0',
+    'awesome-slugify',
```

Comment:
`blessings` is still used in `cli.top`

Reply:
It's already there, in the first position of the list, this just elliminates duplicity.